kilt-modlauncher/build.gradle.kts

```diff
@@ -0,0 +1,10 @@
+repositories {
+    mavenCentral()
+    maven("https://maven.neoforged.net/releases")
+    maven("https://maven.fabricmc.net/")
+}
+
+dependencies {
+    compileOnly("cpw.mods:securejarhandler:3.0.4")
+    compileOnly("net.fabricmc:fabric-loader:0.14.9")
```

Comment:
These should use the versions provided in Kilt's gradle.properties.

Reply:
I thought I fixed those, didn't push my bad